### ISSUE TITLE
Make it possible to search for non-paid promotions in mod actions

### ISSUE THREAD

**Issue by nonamethanks** [r888888888/danbooru#3841]
This is a minor annoyance, but currently the only way to search for user level promotions that are not done automatically is to do a lot of CTRL+F work and guess-work (in cases where someone is promoted to gold by a mod+, such as what happened to me for example). See [this page](https://danbooru.donmai.us/mod_actions?commit=Search&search%5Bcategory%5D=7&utf8=%E2%9C%93&limit=1000). It would be great to have a way to search for promotions that did not happen via payment, to monitor them given they're extremely much rare than normal upgrades to gold/platinum and tend to be swarmed by these (7 out of 1000, in the above example). 

In short, split the "user level" option in the dropdown mod actions menu into "user account upgrade" and "user level \<insert verb here\>".

**Comment by evazion** [comment]
@r888888888 reopening because the fix script (`script/fixes/054_recategorize_user_level_mod_actions.rb`) still needs to be run.

**Comment by r888888888** [comment]
Script has been run.
